### PR TITLE
Use Statistic Initior Invocation for TestImportStatistics

### PR DESCRIPTION
**Pull request by JadenW**
second attempt to fix broken TestImportStatistics test

### CODE REVIEW

File: tests/frontend/org/voltdb/TestImportStatistics.java [hunk 1]
```diff
@@ -234,17 +236,58 @@ private void pushDataToImporters(int count, int loops, boolean goSlow) throws Ex
...
+    private static final String PROC_NAME_COL = "PROCEDURE_NAME";
+    private static final String INVOCATIONS_COL = "INVOCATIONS";
+
     private void waitForLogEvents(Client client, int count) throws Exception {
-        //Wait 40 sec to get out of backpressure.
-        long end = System.currentTimeMillis() + 40000;
+        //Wait 60 sec to get out of backpressure.
+        long end = System.currentTimeMillis() + 60000;
+        int successCount = 0;
+        Map<String, Boolean> expectedInovcations = new HashMap<>();
```
Comment (vtkstef) [inline tests/frontend/org/voltdb/TestImportStatistics.java]: some less verbose ways to build a map

```java
Map<String,Boolean> expectedInvocations = ImmutableMap.of(
    "TestImportStatistics$TestStatsProcedure7", false,
    "TestImportStatistics$TestStatsProcedure11", false,
    "log_events.insert", false
);
```

**Comment by akhanzode** [comment]
How is this branch in Jenkins if looking good merge.

File: tests/frontend/org/voltdb/TestImportStatistics.java (hunk 1 above)
Comment (JadenW) [inline tests/frontend/org/voltdb/TestImportStatistics.java]: Thanks. Revised

Comment (vtkstef) [inline tests/frontend/org/voltdb/TestImportStatistics.java]: make it of Map<String,AtomicBoolean>

**Pull request merged**